src/gui/categoryfilterwidget.cpp

```diff
@@ -73,6 +73,7 @@ CategoryFilterWidget::CategoryFilterWidget(QWidget *parent)
     setIconSize(Utils::Misc::smallIconSize());
 #if defined(Q_OS_MAC)
     setAttribute(Qt::WA_MacShowFocusRect, false);
+    setIndentation(0);
```

Comment:
@thalieht actually I faield to find this option yesterday, and thus assumed that subcategories simply do not work on macOS. Now I found it, thanks :)

I changed the indentation to not be disabled when they are off, but doing something like this for all the platforms is the job of a separate pull request indeed.